lib/uswds/rails/engine.rb

```diff
@@ -0,0 +1,18 @@
+module Uswds
+  module Rails
+    class Engine < ::Rails::Engine
+      initializer 'uswds-rails.assets.precompile' do |app|
+        %w(stylesheets javascripts fonts images).each do |sub|
```

Comment:
in my WIP branch(https://github.com/18F/uswds-rails-gem/compare/jy-add-the-assets?expand=1) I've been putting these all in `app` since that is the rails-y path but can change - what do you think?